src/foraging_gui/Dialogs.py

```diff
@@ -1706,3 +1709,92 @@ def _KeepOpen(self):
             # change button color
             self.KeepOpen.setStyleSheet("background-color : none")
             self.KeepOpen.setChecked(False)
+
+
+class PandasModel(QAbstractTableModel):
```

Comment:
Can you provide a docstring for this class?

Reply:
done.